pkg/api/schema/schema.proto

```diff
@@ -372,10 +372,11 @@ message VerifiableSQLEntry {
 	InclusionProof inclusionProof = 3;
 	uint64 DatabaseId = 4;
 	uint64 TableId = 5;
-	string PKName = 6;
+	repeated uint64 PKIDs = 6;
```

Comment:
the type of entry changes, we should get a new id for it to avoid backwards incompatibility at the protobuf level

Reply:
this release will include breaking changes we should address